app/controllers/food_requests_controller.rb

```diff
@@ -0,0 +1,24 @@
+class FoodRequestsController < ApplicationController
+
+  def create
+
+    current_user.food_requests.where("created_at BETWEEN ? AND ?", Date.today, Time.now).each do |i|
+      i.destroy
+    end
+
+    @f = FoodRequest.new(user_id: current_user.id, daily_meal_id: params[:daily_meal_id])
+    if @f.save
+      redirect_to catering_path(params[:watched_catering])
+      flash[:notice] ="win"
+    else
+      redirect_to catering_path(params[:watched_catering])
+      flash[:alert] = "no + #{params[:daily_meal_id]} + #{params[:watched_catering]} + #{current_user.id}"
```

Comment:
Decide whether you want to use string interpolation or concatenate the strings with `+` for consistency.